packages/astro/src/@types/astro.ts

```diff
@@ -245,6 +245,10 @@ export interface AstroGlobal<
 	 * [Astro reference](https://docs.astro.build/en/guides/server-side-rendering/)
 	 */
 	redirect: AstroSharedContext['redirect'];
+	/**
+	 * TODO add documentation
```

Comment:
Are all the TODOs here intentional for this PR?

Reply:
Yes, they are all intentional and I plan to resolve them in a later PR